mobile/src/main/java/com/alexstyl/specialdates/util/Notifier.java

```diff
@@ -89,19 +93,53 @@ public void forDailyReminder(ContactEvents events) {
                 );
 
         String title = NaturalLanguageUtils.joinContacts(context, events.getContacts(), 3);
-        String fullText = TextUtils.join(", ", events.getContacts());
 
         NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
                 .setSmallIcon(R.drawable.ic_stat_contact_event)
                 .setContentTitle(title)
                 .setLargeIcon(largeIcon)
-                .setStyle(new NotificationCompat.BigTextStyle().bigText(fullText))
                 .setVisibility(NotificationCompat.VISIBILITY_PRIVATE)
                 .setAutoCancel(true)
-                .setContentText(fullText)
                 .setContentIntent(intent)
+                .setNumber(events.getContactCount())
                 .setColor(context.getResources().getColor(R.color.main_red));
 
+        // if event.getContacts() > 1 then use InboxStyle, otherwise the BigTextStyle like here:
+        if (events.getContacts().size() == 1) {
+            int age = events.getContacts().get(0).getBirthday().getAgeOnYear(events.getDate().getYear());
```

Comment:
You are assuming in this point that all ContactEvents received are birthdays. If `forDailyReminder()` gets called only for contact Namedays, you will be displaying a notification as if it's that contact's birthday which is wrong.

You can see a similar snippet of code of how to display specific information about the event on the [ContactEventView](https://github.com/alexstyl/Memento-Namedays/blob/master/mobile/src/main/java/com/alexstyl/specialdates/upcoming/ui/ContactEventView.java#L51) class.